src/server.h

```diff
@@ -3346,7 +3346,9 @@ int calculateKeySlot(sds key);
 
 /* kvstore wrappers */
 int dbExpand(serverDb *db, uint64_t db_size, int try_expand);
+int dbExpand_CreateIfMissing(serverDb *db, uint64_t db_size, int try_expand);
 int dbExpandExpires(serverDb *db, uint64_t db_size, int try_expand);
+int dbExpandExpires_CreateIfMissing(serverDb *db, uint64_t db_size, int try_expand);
```

Comment:
dbExpand is only used in RDB loading and in DEBUG POPULATE. dbExpandExpires is only used in RDB loading.

I think we can imply always do create-if-missing in dbExpand and dbExpandExpires and in the kvstore kvstoreDictExpand and other functions. No code needs the "don't expand if mising" behaviour. We don't need to add new functions.

Side note: We don't use the mixed `camelCase_SnakeCase` naming style for the "db" functions.